webapp/store/views.py

```diff
@@ -14,6 +14,23 @@ def store():
     return render_template('store/store.html')
 
 
+@jaasstore.route('/q/<query>/')
```

Comment:
This route will lead to 500s if someone searches for `my/bundle`

Reply:
Well spotted, I'll see how we handle this in blues browser.

Reply:
Fixed. I've made this url structure work like it does on jujucharms.com, but it could probably be improved if/when we don't have to have equal functionality/compatibility with jujucharms.com.